ajax_select/__init__.py

```diff
@@ -13,6 +13,9 @@
 from django.utils.encoding import force_text
 from django.utils.html import escape
 from django.utils.translation import ugettext_lazy as _
+from django.utils.module_loading import autodiscover_modules
```

Comment:
so this fails in django 1.5 / 1.6  It would be acceptable to not offer autodiscover for django 1.5/6 and do the import inside the function.

But when is autodiscover used ? I need some documentation to see how it works.